src/articles/SubscribeSearchButton.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import * as s from "./SubscribeSearchButton.sc";
+import useSelectInterest from "../hooks/useSelectInterest";
+import { toast } from "react-toastify";
+import SubscribeToEmailNotifications from "./SubscribeToEmailNotifications";
+
+export default function SubscribeSearchButton({ api, query }) {
+  const { subscribedSearches, removeSearch, subscribeToSearch } =
+    useSelectInterest(api);
+
+  const [textButton, setTextButton] = useState("");
+  const [isSubscribedToSearch, setIsSubscribedToSearch] = useState(false);
```

Comment:
why initialize with false? why not leave it uninitialized, as you do in the `SubscribeToEmailNotifications` for a similar variable?